lib/DailyDouble.js

```diff
@@ -1,17 +1,17 @@
 class DailyDouble {
-  constructor(id, wager) {
-    this.id = id;
-    this.wager = wager;
+  constructor() {
+    this.wager;
+    this.highRange;
   }
 
   validateWager(wager, boardValue, player) {
     if (boardValue < player.score) {
-      let highRange = player.score;
+      this.highRange = player.score;
     } else {
-      let highRange = boardValue;
+      this.highRange = boardValue;
     }
```

Comment:
This could potentially be a good spot for a ternary:

`this.highRange = (boardValue < player.score) ? player.score : boardValue;`